tml_decoder/models/vec2text_model.py

```diff
@@ -0,0 +1,29 @@
+from typing import Iterable
+from tml_decoder.models.abstract_model import AbstractLabelModel
+from vec2text import invert_embeddings, load_corrector, models
+from sentence_transformers import SentenceTransformer
+import torch
+
+
+class Vec2TextModel(AbstractLabelModel):
+    def __init__(self, num_steps=10, device="mps"):
+        self.corrector = load_corrector("gtr-base")
+        self.encoder = SentenceTransformer(
+            "sentence-transformers/gtr-t5-base", device=device
```

Comment:
Are we sure that the weights of this model are the same ones that the vec2text was trained on?

Reply:
Yes, `vec2text` supports only two encoders and `gtr-base` is one of them. It's in their docs as well.